specification.md

```diff
@@ -4225,7 +4130,7 @@ Different states now can use this retry strategy defined. For example:
 {
 "onErrors": [
   {
-    "error": "Inventory service timeout",
+    "condition": "${ .item.error == 'Inventory service timeout' }",
```

Comment:
Why does the item property all of sudden have error now? It would make more sense to always query against the previously mentioned "exceptions" array. no?

Reply:
if you want to keep it like that (which i think we probably should not) the jq query would be:

.item | .error == "Inventory service timeout"

(and the quotes need to be escaped)